src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterChecksumGenerator.java

```diff
@@ -10,7 +9,7 @@
     private LetterChecksumGenerator() {
     }
 
-    public static String generateChecksum(LetterRequest letter) {
+    public static String generateChecksum(Object letter) {
```

Comment:
Should it be `Serializable` instead? Not strong about it since `serialize` has `Object` signature